### ISSUE TITLE
Wrong solver for sklearn.linear_model.LogisticRegression

### ISSUE THREAD

**Issue by jolars** [Banana1530/SAGA#1]
I noticed you are using `liblinear` in your call to `sklearn.linear_model.LogisticRegression` but really should be using `saga` instead. It is a separate issue, but I also don't think it's entirely fair to run the scikit-learn implementation in parallel whilst glmnet is running on a single thread.
